### ISSUE TITLE
What parameters should be given in user name and password columns

### ISSUE THREAD

**Issue by Rchamarthi14** [expovin/NPLdapInport#2]
Hi,

I have installed the latest version and entered the parameters in http://localhost like server, base DN and filters. in the next columns, what parameters should be given as user name and password.  Should user ID has admin access?? please advise

**Comment by expovin** [comment]
No, it is not needed the admin access, but just the reading rights. You need to put the username in the LADP format (Common Name, Organizational Unit, Domain Component) es. CN=Vincenzo Esposito, OU=Qliktech, DC=com

Let me know if you'll read the LDAP in this way

**Comment by Rchamarthi14** [comment]
I could not able to read the ldap.. ldap://phsi.primehealthcare.com.. this is the way I have given server name and given user name as CN=userID, dc=..,dc=..,dc=..  but getting invalid credentials error in the command prompt. Pls help

**Comment by expovin** [comment]
Are you able to read it using a third party tool? I often use Softer LDAP Browser to access LDAP. If you are able to use it you should also albe to use this extension. 
Take care if your LDAP accept only secured connections (LDAPs). This version do not support LDAPs

**Comment by Rchamarthi14** [comment]
I just used ldap server name and filter details in the local host URL.  It is giving me invalid credentials error when I supplied user name and password. We have been using same server name and filters in Qliksense User directory connector and its working there without supplying user name and password.

I have also tried same here in http://localhost:3000 URL without giving user name and password but its not working there. I have given my user ID and password as I am part of LDAP but then also it is not working. User ID and password should have any previleges other than part of LDAP. Please advise
